src/sentry/utils/jwt.py

```diff
@@ -0,0 +1,64 @@
+"""Common handling of JWT tokens.
+
+This is an attempt to have all the interactions with JWT in once place, so that we have once
+central place which handles JWT in a uniform way.
+"""
+
+from typing import Mapping, Union
+
+import jwt
+from jwt import DecodeError
+
+__all__ = ["peek_claims", "decode", "encode", "authorization_header", "DecodeError"]
+
+
+def peek_claims(token: str) -> Mapping[str, str]:
+    """Returns the claims (payload) in the JWT token without validation.
+
+    These claims can be used to look up the correct key to use in :func:`decode`.
+    """
+    return jwt.decode(token, verify=False)
+
+
+def decode(token: str, key: bytes, verify_aud: bool = True) -> Mapping[str, str]:
+    """Returns the claims (payload) in the JWT token.
+
+    This will raise an exception if the claims can not be validated with the provided key.
+
+    :param verify_aud: By default if the claims in the token contain an audience ("aud")
+       then we would have to provide the audience at decode time to verify it matches.  If
+       your claims include an audience claim you can use this to ignore it.
+    """
+    options = dict()
+    if not verify_aud:
+        options["verify_aud"] = False
+    return jwt.decode(token, key, options=options)
+
+
+def encode(
+    payload: Mapping[str, str],
+    key: bytes,
+    *,
```

Comment:
We don't user the `..., *, ...` pattern anywhere else in this repo.

Reply:
Is this just an observation or a request not to start using it?  I generally like it to force naming optional arguments.

Reply:
It's valid python so I think you can keep it. If you do I'd suggest adding a comment like:
```
*,  # Force naming optional arguments.
```
because this will be the first time some developers see the pattern.